docs/topics/Permissions.md

```diff
@@ -1,8 +1,10 @@
 # Permissions
 
-Permissions in Discord are a way to limit and grant certain abilities to users. A set of base permissions can be configured at the guild level for different roles, when these roles are attached to users they grant or revoke specific privileges within the guild. Along with the global guild-level permissions, Discord also supports role overwrites which can be set at the channel level allowing customization of permissions on a per-role, per-channel basis.
+Permissions in Discord are a way to limit and grant certain abilities to users. A set of base permissions can be configured at the guild level for different roles. When these roles are attached to users, they grant or revoke specific privileges within the guild. Along with the guild-level permissions, Discord also supports permission overwrites that can be set at the channel-level. These overwrites allow for the customization of permissions on a per-role or per-member, per-channel basis.
```

Comment:
I think this may be slightly better worded as:

```
Along with the guild-level permissions, Discord also supports permission overwrites that can be set at the channel, role, and user level. These overwrites allow for the granular customization of permissions.
```

Reply:
Hmmm. My wording makes the distinction between permissions that are applied to roles at the guild-level versus permissions that are applied to roles and members at the channel-level. I'm not sure it makes sense to call a permission "role-level" or "user-level" with that in mind. Listing "channel, role, and user level" makes it sound like you can apply a permission to each, but you can't apply a permission to just a channel. How about:

```
Permissions in Discord are a way to limit and grant certain abilities to users. A set of base permissions can be configured at the guild level for different roles. When these roles are attached to users, they grant or revoke specific privileges within the guild. Along with the guild-level permissions, Discord also supports permission overwrites that can be assigned to individual guild roles or guild members on a per-channel basis.
```

Reply:
Yeah—I like that better. Good call 👍